docker-qgis/entrypoint.py

```diff
@@ -104,22 +99,29 @@ def _call_qfieldsync_packager(project_filename: Path, package_dir: Path) -> str:
         vl_extent_wkt = vl_extent.asWktPolygon()
         vl_extent_crs = project.crs()
 
-    logging.info(
+    logger.info(
         f"Project area of interest is `{vl_extent_wkt}` in `{vl_extent_crs}` CRS."
     )
 
     attachment_dirs, _ = project.readListEntry("QFieldSync", "attachmentDirs", ["DCIM"])
-    offline_editing = QgsOfflineEditing()
+    if offliner_type == OfflinerType.QGISCORE:
+        offliner = QgisCoreOffliner()
+    elif offliner_type == OfflinerType.PYTHONMINI:
+        offliner = PythonMiniOffliner()
+    else:
+        raise NotImplementedError(f"Offliner type {offliner_type} is not supported.")
```

Comment:
Could we instead of raising an Error just log and set the default value?
```suggestion
    if offliner_type == OfflinerType.QGISCORE:
        offliner = QgisCoreOffliner()
    elif offliner_type == OfflinerType.PYTHONMINI:
        offliner = PythonMiniOffliner()
    else:
        logger.error(f"Offliner type {offliner_type} is not supported.")
        offliner = QgisCoreOffliner()
```

Reply:
I would argue we should keep the `raise NotImplementedException` for enumerations, makes it easier for developer.